docker/services/fcrepo/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM tomcat:8.0-jre8
+
+MAINTAINER Yinlin Chen "ylchen@vt.edu"
+
+RUN apt-get update && apt-get -y upgrade
+
+# Install Fedora4
+ARG FEDORA_VERSION=4.7.5
+ARG FEDORA_TAG=4.7.5
+ARG FedoraConfig=
+ARG ModeshapeConfig=file-simple
+ARG JDBCConfig=
+
+ENV CATALINA_HOME /usr/local/tomcat
+ENV PATH $CATALINA_HOME/bin:$PATH
+
+RUN mkdir /mnt/ingest
+
+RUN sed -i '$i<role rolename="fedoraUser"/>$i<role rolename="fedoraAdmin"/>$i<role rolename="manager-gui"/>$i<user username="testuser" password="password1" roles="fedoraUser"/>$i<user username="adminuser" password="password2" roles="fedoraUser"/>$i<user username="fedoraAdmin" password="secret3" roles="fedoraAdmin"/>$i<user username="fedora4" password="fedora4" roles="manager-gui"/>' /usr/local/tomcat/conf/tomcat-users.xml
```

Comment:
Does this mean all these usernames and passwords are hard-coded?

Reply:
Yes. for now. These three Fedora user accounts are documented in README.